src/misc/lv_anim.c

```diff
@@ -483,9 +482,25 @@ lv_anim_t * lv_anim_custom_get(lv_anim_t * a, lv_anim_custom_exec_cb_t exec_cb)
     return lv_anim_get(a ? a->var : NULL, (lv_anim_exec_xcb_t)exec_cb);
 }
 
+uint32_t lv_anim_resolve_speed(uint32_t speed_or_time, int32_t start, int32_t end)
+{
+    /*It was a simple time*/
+    if((speed_or_time & 0x80000000) == 0) return speed_or_time;
```

Comment:
It is more a nit but, what means the hex value? Is there a definition for it? Can we use it instead of this magic val?

Reply:
So thank you for mentioning.  This nits are important to improve the quality. 

Just added a define for it.